Cargo.toml

```diff
@@ -93,6 +93,7 @@ bytes = { version = "0.4.10", features = ["serde"] }
 stream-cancel = "0.4.3"
 prometheus = "0.4.2"
 hyper = "0.12.35"
+hyper-openssl = "0.7"
```

Comment:
since we are sticking with openssl, we should probably update the rest of our hyper based crates to using this.